app/initialize.js

```diff
@@ -1,6 +1,8 @@
 // Note: there's no longer any need to wait for DOMContentReady here as we
-// not use jQuery.ready() in order to handle this logic directly in index.html
-// prior to attempting to load this 'initialize' CommonJS module.
+// now use jQuery.ready() in order to handle the DOM ready logic directly within
+// index.html prior to attempting to load this 'initialize' CommonJS module.
+// This means that you can count on the document being fully ready by the
+// time this file (and all its dependencies) get loaded.
 
-// Initialize your main JS module here. For example:
+// Initialize/require your main JS module(s) here. For example:
```

Comment:
Super-minor thing, but I'd recommend only having one main module for dataviz projects.